src/stores/simulation.ts

```diff
@@ -32,12 +32,14 @@ export const resources: Writable<Resource[]> = writable([]);
 
 export const fetchingResources: Writable<boolean> = writable(false);
 
-export const fetchingResourcesExternal: Writable<boolean> = writable(false);
+export const fetchingResourcesExternal: Writable<boolean> = writable(true);
```

Comment:
was this meant to be initialized to true?

Reply:
But I did intentionally set it to true before so that the Row wouldn't start out thinking that external resources were ready to be accessed. Basically I think we'd need an "externalResourcesInitialized" or something of the sort to signal that we've started that process if we didn't default to true?

Reply:
Added a comment as to why it needs to be true for now